docs/doxygen_guide.md

```diff
@@ -98,24 +98,24 @@ so that Doxygen can copy the function signature to custom function pages.
     /* @[declare_mylibrary_myfunction] */
     ```
 
-1. Please copy the all of the template pages to your library's doc/doxygen folder.
-For our CI to detect files correctly, please keep the folder name as *doc/doxygen*
+1. Please copy the all of the template pages to your library's docs/doxygen folder.
+For our CI to detect files correctly, please keep the folder name as *docs/doxygen*
 and all of the files names the same.
 
     ```console
-    cp -R <csdk_root>/doc/doxygen_templates <library_root>/doc/doxygen
+    cp -R <csdk_root>/docs/doxygen_templates <library_root>/docs/doxygen
```

Comment:
I think this should be `config.doxyfile`

Reply:
No this is copying all the files in the directory. Let me add the missing *.